specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,731 @@
+                    },
+                    "404": {
+                        "description": "Not found",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "405": {
+                        "description": "Method not allowed",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "429": {
+                        "description": "Too many requests",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "503": {
+                        "description": "Service unavailable",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "500": {
+                        "description": "Server error",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get Instance Metadata for the VM": {
+                        "$ref": "./examples/getinstancemetadata.json"
+                    }
+                }
+            }
+        },
+        "/attested/document": {
+            "get": {
+                "operationId": "getAttested",
+                "description": "Get Attested Data for the Virtual Machine.",
+                "parameters": [
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "required": true,
+                        "type": "string",
+                        "description": "This is the version of API to invoke."
+                    },
+                    {
+                        "name": "nonce",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "10-digit random number."
+                    },
+                    {
+                        "name": "Metadata",
+                        "in": "header",
+                        "type": "string",
+                        "enum": [
+                            "true"
+                        ],
+                        "description": "This must be set to 'true'.",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/attested"
+                        }
+                    },
+                    "400": {
```

Comment:
Please replace all the negative response codes with only one response code. `"default"`.

```json5
"default": {
  "description": " Error response",
  "schema": {
     "$ref": "#/definitions/ErrorResponse"
   }
}
```
This will ensure that the sdk throws an exception for all the negative responses where the deserialized Error is based on the provided ErrorResponse schema.

Reply:
Please do this for all the operations.

Reply:
Done.